templates/internet-of-things/smart-city.html

```diff
@@ -474,7 +474,7 @@ <h2>Top-rated extended security. Best-in-class isolation.</h2>
     <div class="col-6 u-vertically-center u-align--center u-hide--small">
       {{ image (
         url="https://assets.ubuntu.com/v1/c53664cd-Security_updates.svg",
-        alt="",
+        alt="Three shield illustration with 10 years written on the middle shield, and green ticks on the other two",
```

Comment:
The image description doesn't add any detail that an assistive tech user can't learn from the associated text to the right, so we don't need this.

```suggestion
        alt="",
```